desktop/cmp/dash/canvas/DashCanvasModel.js

```diff
@@ -74,6 +84,19 @@ export class DashCanvasModel extends HoistModel {
     ref = createObservableRef();
     /** @member {boolean} */
     scrollbarVisible;
+    /** @returns {boolean} */
+    isLoadingState;
+    /** @returns {boolean} - true if any viewModels are in the middle of auto-sizing */
+    get isAutoSizing() {
+        return some(this.viewModels, 'isAutoSizing');
+    }
+    /** @returns {Object[]} */
+    @computed get rglLayout() {
```

Comment:
I think we want `@computed.struct` here since you are always returning a new array